### DISCUSSION TITLE
Post-Doom init script (colorscheme)

### DISCUSSION THREAD

**Discussion by J-Kappes** [NTBBloodbath/doom-nvim#433] [Q&A] [answered]
When I put `vim.cmd("colorscheme dracula")` in my `config.lua` I get an error message saying `Cannot find color scheme 'dracula'`. Apparently the colorscheme isn't loaded yet. How can I set the colorscheme automatically on startup?

**Comment by kanzaki1201** [comment, marked as answer]
I struggled on this for a while too but turns out it is just a simple `doom.colorscheme = "<name of theme>"`
For example: `doom.colorscheme = "tokyonight"`
